### ISSUE TITLE
Naz and Chaotic frontend issues 2

### ISSUE THREAD

**Issue by nnamdiarimah** [Rug-Zombie/rug-zombie-frontend#119]
- [ ] https://github.com/Rug-Zombie/rug-zombie-frontend/issues/118
- [x] https://github.com/Rug-Zombie/rug-zombie-frontend/issues/120
- [x] https://github.com/Rug-Zombie/rug-zombie-frontend/issues/121
- [x] https://github.com/Rug-Zombie/oblivion/issues/54
Same as last time:
When starting a new task make sure your do the following:
- switch to master branch `git checkout master`
- pull the most recent changes `git pull`
- create your new branch `git checkout -b BRANCH_NAME`

This makes things a lot easier later when merging your work. Once a task is complete you can push and create a pull request where I'll give feedback. Once it's ready and merged we'll mark it as complete.
